packages/snap-preact-components/src/components/Molecules/VariantSelection/VariantSelection.tsx

```diff
@@ -0,0 +1,212 @@
+/** @jsx jsx */
+import { h, Fragment } from 'preact';
+import { observer } from 'mobx-react';
+import { jsx, css } from '@emotion/react';
+import classnames from 'classnames';
+import { Theme, useTheme, CacheProvider } from '../../../providers';
+import { defined } from '../../../utilities';
+import { ComponentProps, ListOption, StylingCSS } from '../../../types';
+import type { VariantSelection as VariantSelectionType } from '@searchspring/snap-store-mobx';
+import { List, ListProps } from '../List';
+import { Grid, GridProps } from '../Grid';
+import { Swatches, SwatchesProps } from '../Swatches';
+import { Option, Select, SelectProps } from '../Select';
+
+const CSS = {
+	variantSelection: () => css({}),
+};
+
+export const VariantSelection = observer((properties: VariantSelectionProps): JSX.Element => {
+	const globalTheme: Theme = useTheme();
+
+	const props: VariantSelectionProps = {
+		// default props
+		type: 'select',
+		// global theme
+		...globalTheme?.components?.variantSelection,
+		// props
+		...properties,
+		...properties.theme?.components?.variantSelection,
+	};
+
+	const { type, selection, hideTitle, disableStyles, className, style } = props;
+
+	const subProps: VariantSelectionSubProps = {
+		select: {
```

Comment:
This component doesn't seem to handle "disabled" display like the others do, but is it possible? If not, is that a deal breaker for this being an option here?